nibabel/nifti1.py

```diff
@@ -723,6 +723,11 @@ def set_data_shape(self, shape):
         If ``ndims == len(shape)`` then we set zooms for dimensions higher than
         ``ndims`` to 1.0
 
+        Nifti1 images can have up to seven dimensions. For Nifti surface files,
```

Comment:
Nice - but - can you clean up the appearance of the links in the output by using ReST links, defined at the bottom of the docstring?